app-text/grcompiler/grcompiler-5.2.1.recipe

```diff
@@ -0,0 +1,63 @@
+SUMMARY="SIL Graphite compiler"
+DESCRIPTION="The SIL Graphite compiler builds a Graphite enabled font \
+from a smart font description, written in GDL (Graphite Description Language)\
+and a TrueType font by adding extra TrueType tables to it which the\
+ Graphite engine can interpret."
+HOMEPAGE="http://graphite.sil.org/"
+COPYRIGHT="2002 SIL International"
+LICENSE="GNU GPL v2"
+REVISION="1"
+SOURCE_URI="https://github.com/silnrsi/grcompiler/archive/refs/tags/v$portVersion.tar.gz"
+SOURCE_FILENAME="grcompiler-$portVersion.tar.gz"
+CHECKSUM_SHA256="a125e9d2746638a538e59af8ece2268fda078572f50e82f82d6f95ccb08ed890"
+
+ARCHITECTURES="!x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+commandBinDir=$binDir
+commandSuffix=$secondaryArchSuffix
+if [ "$targetArchitecture" = x86_gcc2 ]; then
+	commandSuffix=
+	commandBinDir=$prefix/bin
+fi
+
+PROVIDES="
+	grcompiler$secondaryArchSuffix = $portVersion
+	cmd:grcompiler$commandSuffix = $portVersion
+	cmd:gdlpp$commandSuffix = $portVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libiconv$secondaryArchSuffix
+	lib:libicui18n$secondaryArchSuffix >= 66
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:libiconv$secondaryArchSuffix
+	devel:libicui18n$secondaryArchSuffix >= 66
+	"
+BUILD_PREREQUIRES="
+	cmd:gcc$secondaryArchSuffix
+	cmd:make
+	cmd:cmake
+	cmd:pkg_config$secondaryArchSuffix
+	"
+
+BUILD()
+{
+	mkdir -p build
+	cd build
+	cmake .. -DCMAKE_BUILD_TYPE=Release -DCMAKE_INSTALL_PREFIX=$prefix
+	make
```

Comment:
Simplify, use cmakeDirArgs and $commandBinDir
```diff
-	mkdir -p build
-	cd build
-	cmake .. -DCMAKE_BUILD_TYPE=Release -DCMAKE_INSTALL_PREFIX=$prefix
-	make
+	cmake -Bbuild -S. -DCMAKE_BUILD_TYPE=Release \
+		$cmakeDirArgs \
+		-DCMAKE_INSTALL_BINDIR=$commandBinDir
+	make -C build $jobArgs
``
	make

Reply:
updated